src/bin/console/console.cpp

```diff
@@ -211,14 +212,13 @@ int Console::_eval_command(const CommandFunction& func, const std::string& comma
 }
 
 int Console::_eval_sql(const std::string& sql) {
-  SQLQueryPlan plan;
-  hsql::SQLParserResult parse_result;
+  _sql_pipeline = std::make_unique<SQLPipeline>(sql, _explicitly_created_transaction_context);
 
   // Measure the query parse time
   auto started = std::chrono::high_resolution_clock::now();
 
   try {
-    hsql::SQLParser::parse(sql, &parse_result);
+    _sql_pipeline->get_parsed_sql();
   } catch (const std::exception& exception) {
     out("Exception thrown while parsing SQL query:\n  " + std::string(exception.what()) + "\n");
```

Comment:
The exception message is already set now in the SqlPipeline, right? This would produce:
```
Exception thrown while parsing SQL query:
Error while parsing SQL query:
*actual exception message*
```

Reply:
Good point. I guess this can be taken out anyway, depending on whether we execute all in a single try-catch